apps/sample-app/public/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.10/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.10",
+  "version": "0.2",
+  "id": "8142d2d1-4113-411f-868f-8e6e606e8827",
+  "packageName": "sample app",
+  "developer": {
+    "name": "Ananya Barthakur",
```

Comment:
>   "name": "Ananya Barthakur", [](http://example.com/codeflow?start=1&length=30)

This may not be the right name to put here. Consider "Microsoft"?